rancher2/schema_service_account_token.go

```diff
@@ -0,0 +1,100 @@
+package rancher2
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+//Schemas
+
+func serviceAccountTokenFields() map[string]*schema.Schema {
+	s := map[string]*schema.Schema{
+		"password": {
```

Comment:
Add `Description: <description>` to password, username, temp token, and temp token id fields